### DISCUSSION TITLE
After update 1.35.0 edges appear below workflow nodes

### DISCUSSION THREAD

**Discussion by AlessandroPomponio** [bcakmakoglu/vue-flow#1503] [Q&A] [answered]
Hello,

We are using vue-flow for rendering graphs of workflows, their components, and the edges that connect them. For clarity, in the screenshots that follow:
- Workflows are going to be the green nodes
- Components are going to be the white nodes
- Edges are in grey

Up until version [v1.34.1](https://github.com/bcakmakoglu/vue-flow/releases/tag/v1.34.1), when we were rendering the workflow for measuring properties of nano-porous materials, it would appear like this (see for yourself at https://registry.st4sd.res.ibm.com/registry-ui/experiment/nanopore-geometry-experiment/viewcanvas):

<img width="2032" alt="image" src="https://github.com/bcakmakoglu/vue-flow/assets/10339005/de36f06b-72f0-479b-86c6-90a18a780833">

This is working as we expect it to, with the edges being on top of the workflow.

From version [v1.35.0](https://github.com/bcakmakoglu/vue-flow/releases/tag/v1.35.0), however, we see the following:

<img width="2032" alt="image" src="https://github.com/bcakmakoglu/vue-flow/assets/10339005/628d14b3-2dc6-4dd3-96e7-37c3cb110b4a">

The edges now appear **below** the workflow nodes.

---

By comparing the generated HTML, I noticed that the `z-index` that is assigned to the edges has changed from 1 (old version, on the left) to 0 (new version, on the right):

<img width="1421" alt="image" src="https://github.com/bcakmakoglu/vue-flow/assets/10339005/ed6ac8bb-73ef-448c-ba8f-3a0293146130">

Was this intended? Did we miss anything? Thank you very much in advance.

**Comment by bcakmakoglu** [comment, marked as answer]
Thanks for reporting this. It's an issue introduced because of some memoization that was added.
It'll be fixed in the next release :)

**Reply by AlessandroPomponio** [reply]
Great to hear, thank you very much again!

**Reply by bcakmakoglu** [reply]
Fixed with `1.38.0`
